lib/service.js

```diff
@@ -64,7 +69,12 @@ module.exports = function(repo, manifest) {
 
   function *enqueueToServices(msg) {
     for (var subscriber of _this.repo.getSubscribersOf(msg)) {
-      yield subscriber.enqueueInput(msg)
+      if (msg.deliverInMillis) {
+        var dueTime = new Date().getTime() + msg.deliverInMillis
```

Comment:
You should replace `new Date().getTime()` with the static Date method meant for that: `Date.now()`